draft-ietf-tsvwg-dtls-over-sctp-bis.md

```diff
@@ -504,13 +504,13 @@ terminated and the associated keying material discarded.
 
    The DTLS handshakes MUST use mutual authentication.
 
-## Renegotiation and KeyUpdate
+## Renegotiation and Key Update
 
    DTLS 1.2 renegotiation enables rekeying (with ephemeral Diffie-
    Hellman) of DTLS as well as mutual reauthentication and transfer of
    revocation information inside an DTLS 1.2 connection. Renegotiation
    has been removed from DTLS 1.3 and partly replaced with
-   post-handshake messages such as KeyUpdate. The parallel DTLS
+   post-handshake messages such as Key Update. The parallel DTLS
```

Comment:
This sentence references the actual message name, thus further reformulation needed.